### PR TITLE
Important improvement of the tutorials

### PR DESCRIPTION
**Pull request by valentina-s**

### CODE REVIEW

**Comment by lsetiawan** [comment]
I love the message!!!!

File: text.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ dont know how to leave. Heeeelp!
```
Comment (lsetiawan) [inline text.txt]: it's `:wq`

**Comment by amarburg** [comment]
It doesn't compile on my machine  :-(

**Comment by lsetiawan** [comment]
Closing this demo PR from master. Thanks Valentina! :+1:

**Pull request closed without merging**